.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+# Why
+
+*A summary of what problem this pull request is trying to solve. **Always** include a #reference to an existing issue, which should provide a more detailed explanation of the problem, as well as any discussion about the nature of the problem.*
+
+# What
+
+*A list of the changes included in this pull request. A checkbox list is probably a good choice here, and the boxes should be checked off as tasks are completed. This section may be omitted if the Why section provides enough information and only a few changes were made (eg, for simple bug reports).*
+
+**Always highlight breaking changes or any changes that effect the behaviour of user facing components of the application**
+
+- [ ] Put your changes in a list like this
+- [x] Then you can check them off like this!
+
+*Also, include screenshots if you can!*
+
+# How
+
+*Describe in detail how you solved the above problem. This is not a replacment for documentation, but should give people context and a high-level understanding of your solution before/without reading your code.*
+
+# Outstanding Questions & Concerns
+
+*Any questions or concerns you were unable to address while implementing this solution. For example, "I've implemented heat vision resistance, but do we expect to be attacked by superman?"
+
+If you find that this section is getting long, or the questions and concerns have a significant impact on the quality of this solution, consider making this a ["draft"](https://github.blog/2019-02-14-introducing-draft-pull-requests/) pull request to ensure that it is not merged until it is ready*
+
+*This may be another place where a checklist is a good choice, so people can see which questions have been resolved*
+
+# Next Steps
+
+*Optional. This section is to highlight any ideas you have for how this feature could be extended in the future.*
+
+*You should also consider adding issues for these next steps*
+
+# Security & Accessibility
+
+*Address how this change will improve/lessen the security of the application - for example, it improves HTML sanatization or opens a new attack surface. Always explain so that developers without a security background can understand. Similarly, address how this change will impact the accessibility of the application. Be sure to address both physical accesibility (eg, making the website work better with screenreaders) and technical accesibility (eg, making the website load faster for people with marginal internet). If you're not sure what to put here, don't worry! Just explain what you can, and make sure to note that you think this issue needs to be reviewed with security & accesibility in mind*
```

Comment:
It could be worth splitting the bit about the two types of accessibility into its own line and also making each one a bullet point so they pop out.

Reply:
Splitting this up seems like a good idea - I was trying to find places to minimize the length of the template, but this might be overdoing it